src/job-location.js

```diff
@@ -0,0 +1,26 @@
+import { getName } from 'country-list';
+
+export default class JobLocation {
+  constructor(srLocation) {
+    this.city = srLocation.city;
+    this.region = srLocation.region;
+    this.countryCode = srLocation.country;
+  }
+
+  get country() {
+    return getName(this.countryCode);
+  }
+
+  get citySlug() {
+    return encodeURIComponent(this.city.toLocaleLowerCase());
+  }
```

Comment:
Hmm.. that's fair. Although, we still out to URI-encode it because some city names have non-ASCII characters. E.g. "Gdańsk".

I'll swap out the lower case with a kebab case function. Currently, the only city that'll be affected by that (i.e. have a different URI than it has today) will be "New York" (as it'll become `new-york` rather than `new%20york`).

Ideally, URIs/URLs should be stable and avoid changing, but I think it's Ok in this instance. These URIs are invisible to most people and I suspect the effect on Google and co. is virtually nothing.